vulcan/lib/server/data/status1.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This may not be the most accurate format, just what I'm theorizing right now. I'm assuming the server will return status with steps as a Hash, like CWL format.

Reply:
That's totally fine.  We'll work with this if it's easier.  I can work the backend to fit any shape you want, as long as the correct information is captured by _some_ means.

Reply:
Yeah, I'm not sure right now if this is easier, it's just that the UI needs to re-map the original steps from the CWL from a Hash into an Array. But for subsequent steps, I think the options are either use a Hash (like CWL), or just provide the step `name` in each Hash in the Array. Agree basically we just have to send the step name data somehow...easier on the UI if we keep things in an Array of Steps so there's less manipulation.

Reply:
Then that's totally fine.  _you tell me_ what the UI would prefer and I can make it happen.  I'm ok with complexity in the data transformation on the ruby side because frankly, _testing complexity in ruby is easier than javascript_.